package.json

```diff
@@ -72,7 +72,7 @@
     "@ngrx/store": "6.1.2",
     "@ngrx/store-devtools": "6.1.2",
     "core-js": "^2.5.4",
-    "geteventlisteners": "^1.0.6",
+    "geteventlisteners": "file:./external/geteventlisteners/geteventlisteners-1.0.6.tgz",
```

Comment:
Updates made to geteventlisteners to avoid es6 syntax:
1. Removed usage of default parameters - Added check `if (param === undefined) param = false;` as substitute
2. Removed usage of object property shorthand, i.e. `{ property }` to `{ property: property }`
3. Changed use of `let` keyword to `var`